packages/edit-site/src/components/layout/style.scss

```diff
@@ -38,13 +38,13 @@
 }
 
 .edit-site-layout__canvas {
-	background: $white;
```

Comment:
Don't you think we should add a fallback background for the pages that may not set any (list pages...)

Reply:
This was causing a visual artefact in browse mode that was important to eliminate. I added a new style in https://github.com/WordPress/gutenberg/pull/44770/commits/20784db23906a4a99406710aca87d81eca14ae36 to handle list views.